### PR TITLE
explain ~

### LINKED ISSUE #9119: Is the ~ necessary before static files in wwwroot like /images/banner1.svg?
Using the ~ in my project as shown in ['Serve files inside of web root'](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-2.1&tabs=aspnetcore2x#serve-files-inside-of-web-root) was causing a 404 to display- when I removed it and only had the equivalent of /images/banner1.svg, the code worked fine.  Is the ~ necessary in ASP.NET Core 2.1?  I saw on the samples there is only a sample for 1.x with no index.hmtl and I also saw in the 'Get Started - Create a Web API' that the "site.js" file is not referenced with a ~ for the index.html file so it seems like it is not needed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d47f38e2-bc36-cfb8-b0cc-17a534689f8b
* Version Independent ID: 3fec6e08-fc99-7a5c-796f-3f2347cad891
* Content: [Static files in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-2.1&tabs=aspnetcore2x)
* Content Source: [aspnetcore/fundamentals/static-files.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/static-files.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #9119

### CODE REVIEW

**Comment by scottaddie** [comment]
@Rick-Anderson These tabs were indeed removed from the approved list, but we shouldn't let that block merging. Merging your changes now.

File: aspnetcore/fundamentals/index.md [hunk 1]
```diff
@@ -71,7 +71,9 @@ The content root is the base path to any content used by the app, such as [Razor
 
 ## Web root
 
-The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files.
+The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files. By default, *wwwroot* is the web root.
```
Comment (dougbu) [inline aspnetcore/fundamentals/index.md]: I'm not sure it matters that much but we normally use "webroot" (one word) in all related doc comments.

File: aspnetcore/fundamentals/index.md [hunk 2]
```diff
@@ -71,7 +71,9 @@ The content root is the base path to any content used by the app, such as [Razor
...
+
+For ASP.NET Core code, the tilde character `~` represents the  web root operator and points to the web root.
```
Comment (dougbu) [inline aspnetcore/fundamentals/index.md]: Do not invent the term "web root operator". A path in a .cshtml that starts with `"~/"` (the slash is required) is referred to as a virtual path.

File: aspnetcore/fundamentals/static-files.md [hunk 3]
```diff
@@ -66,6 +66,8 @@ The parameterless `UseStaticFiles` method overload marks the files in web root a
 
 [!code-cshtml[](static-files/samples/1x/Views/Home/Index.cshtml?name=snippet_static_file_wwwroot)]
 
+In the preceding code, the tilde character `~` represents the ASP.NET Core web root operator. For more information, see [Web root](xref:fundamentals/index#web-root).
```
Comment (dougbu) [inline aspnetcore/fundamentals/static-files.md]: Again, don't invent the term "web root operator" and the `~` on its own doesn't mean anything. A "virtual path" begins with `"~/"`.

File: aspnetcore/fundamentals/index.md [hunk 4]
```diff
@@ -71,7 +71,9 @@ The content root is the base path to any content used by the app, such as [Razor
 
 ## Web root
 
-The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files.
+The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files. By default, *wwwroot* is the web root.
+
+For ASP.NET Core code, the tilde character (`~`) represents the web root operator and points to the web root.
```
Comment (dougbu) [inline aspnetcore/fundamentals/index.md]: Do not invent the term "web root operator". A path in a .cshtml that starts with `"~/"` (the slash is required) is referred to as a virtual path.

And, what is "ASP.NET Core code"? Do you mean "in a `.cshtml` file"?

File: aspnetcore/fundamentals/static-files.md [hunk 5]
```diff
@@ -66,6 +66,8 @@ The parameterless `UseStaticFiles` method overload marks the files in web root a
 
 [!code-cshtml[](static-files/samples/1x/Views/Home/Index.cshtml?name=snippet_static_file_wwwroot)]
 
+In the preceding code, the tilde character (`~`) represents the ASP.NET Core web root operator. For more information, see [Web root](xref:fundamentals/index#web-root).
```
Comment (dougbu) [inline aspnetcore/fundamentals/static-files.md]: Again, don't invent the term "web root operator" and the `~` on its own doesn't mean anything. A "virtual path" begins with `"~/"`.

**Comment by Rick-Anderson** [comment]
@dougbu check and suggest better wording if necessary.

File: aspnetcore/fundamentals/index.md [hunk 6]
```diff
@@ -69,9 +69,11 @@ For more information, see <xref:fundamentals/startup>.
 
 The content root is the base path to any content used by the app, such as [Razor Pages](xref:razor-pages/index), MVC views, and static assets. By default, the content root is the same location as the app base path for the executable hosting the app.
 
-## Web root
+## Web root (webroot)
 
-The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files.
+The webroot of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files. By default, *wwwroot* is the webroot.
+
+For Razor (*.cshtml*) files, the tilde-slash  `~/` points to the webroot. `~/` is referred to as the virtual path of the app.
```
Comment (dougbu) [inline aspnetcore/fundamentals/index.md]: Leave out the second sentence in this paragraph. Nobody says "virtual path of the app".

If you want to mention virtual path (I'm not sure it's necessary), you could say "Paths beginning with `~/` are referred to as virtual paths."

File: aspnetcore/fundamentals/static-files.md [hunk 7]
```diff
@@ -66,6 +66,8 @@ The parameterless `UseStaticFiles` method overload marks the files in web root a
 
 [!code-cshtml[](static-files/samples/1x/Views/Home/Index.cshtml?name=snippet_static_file_wwwroot)]
 
+In the preceding code, the tilde character `~/` points to the virtual path of the app. For more information, see [Web root](xref:fundamentals/index#web-root).
```
Comment (dougbu) [inline aspnetcore/fundamentals/static-files.md]: Stick with "points to the webroot"

**Pull request merged**